styles/styles.css

```diff
@@ -0,0 +1,495 @@
+@import url('https://fonts.googleapis.com/css?family=Archivo+Black|Merriweather:400,400i');
+
+@media screen and (max-width: 767px) {
```

Comment:
I'd recommend putting these media queries at the end of your CSS just in case somthing below doesn't override them.